src/actions/PetitionActions.js

```diff
@@ -54,6 +61,10 @@ export function submittingPetition () {
 
 export function createPetition (petition, dispatch) {
   dispatch(submittingPetition());
+
+  // Wrap the URLs in an object to fit the API needs.
+  petition.links = wrapPetitionLinks(petition);
+
   return petitionRepository.create(petition)
     .then((response) => {
       const resolvedPetition = solveResolvedObjects(petition, response.data);
```

Comment:
I don't think we need this, if we use the `resolve` param when making the API call